app/controllers/warehouse_reports/active_veterans_controller.rb

```diff
@@ -31,6 +31,12 @@ def show
       end
     end
 
+    def destroy
+      @report = report_source.find params[:id]
```

Comment:
To prevent SQL injection, this should be like the following.
```suggestion
      @report = report_source.find params[:id].to_i
```

Reply:
The same applies to the other find calls in this PR